engine/src/render/shadow_atlas/plugin.cpp

```diff
@@ -38,16 +39,17 @@ void cubos::engine::shadowAtlasPlugin(Cubos& cubos)
 
     cubos.system("reserve space for shadow casters")
         .tagged(reserveShadowCastersTag)
-        .call([](ShadowAtlas& atlas, Query<SpotShadowCaster&> casters) {
+        .call([](ShadowAtlas& atlas, Query<SpotShadowCaster&, const Active&> casters) {
             atlas.slots.clear();
             atlas.slotsMap.clear();
             atlas.slots.push_back(
                 std::make_shared<ShadowAtlas::Slot>(glm::vec2(1.0F, 1.0F), glm::vec2(0.0F, 0.0F), -1));
 
             int id = 1;
 
-            for (auto [caster] : casters)
+            for (auto [caster, active] : casters)
             {
+
```

Comment:
Still missing the active check.
```suggestion
if (!active.active)
{
    continue;
}
```